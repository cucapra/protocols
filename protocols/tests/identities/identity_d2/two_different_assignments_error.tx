// ARGS: --verilog identities/identity_d2/identity_d2.v --protocol identities/identity_d2/identity_d2.prot
// RETURN: 101
multiple_assign(1, 1);
multiple_assign(2, 2);
