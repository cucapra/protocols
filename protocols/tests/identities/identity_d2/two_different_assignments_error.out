[0m[1m[38;5;9merror[0m[1m: Thread 0 attempted conflicting assignment to 'a': current=ValueOwned(00000000000000000000000000000001), new=ValueOwned(00000000000000000000000000000010)[0m
  [0m[34mâ”Œâ”€[0m identities/identity_d2/identity_d2.prot:9:3
  [0m[34mâ”‚[0m
[0m[34m9[0m [0m[34mâ”‚[0m   [0m[31mDUT.a := a;[0m
  [0m[34mâ”‚[0m   [0m[31m^^^^^^^^^^^[0m [0m[31mThread 0 attempted conflicting assignment to 'a': current=ValueOwned(00000000000000000000000000000001), new=ValueOwned(00000000000000000000000000000010)[0m

